support-frontend/package.json

```diff
@@ -68,7 +68,9 @@
     "@emotion/babel-preset-css-prop": "^10.0.27",
     "@emotion/cache": "^10.0.29",
     "@emotion/core": "^10.0.27",
+    "@emotion/serialize": "^0.11.16",
```

Comment:
Do we still need this now the SSR has been removed?

Reply:
I believe we do need it, but I will check.

Reply:
Just to confirm, yes we do need it.